app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list[dict], cafe: "Cafe") -> str:
```

Comment:
Don't forget, that u can import annotation from `__future__` to use class as annotation without quotes